docs/reference/fpp-json-dict.md

```diff
@@ -920,7 +920,7 @@ module M {
 
 | Field | Description | Options | Required |
 | ----- | ----------- | ------- | -------- |
-| `deploymentName` | **String** representing the deployment name | **String** | true |
+| `deploymentName` | Fully qualified name of the topology | **String** | true |
```

Comment:
This looks good, but maybe we should keep the form "**String** representing ..." for consistency with the other fields? Or revise the other fields to match.

Reply:
I'll update it for consistency